bin/aursync

```diff
@@ -55,13 +59,19 @@ download() {
 
     declare gz
 
-    parallel --will-cite -X "curl -s --remote-name-all {}"
+    if command_exists aria2c; then
```

Comment:
I'm not too fond of the idea of supporting two methods for parallel downloading. If `-X` and increased jobs makes parallel as effective as aria2c I'd consider parallel, otherwise just stick to aria2c with curl as fallback.